### PR TITLE
267988: Added database permission scripts 

### PR DESCRIPTION
**Pull request by jeevankuduvaravindran**
# **What this PR does / why we need it**:
This PR contains the changes to add database permission scripts to be executed via flux and dockerfile to run the powershell scripts.

[AB#267988](https://dev.azure.com/defragovuk/93c65135-d16b-49d6-a191-4a5313532779/_workitems/edit/267988)

# **Special notes for your reviewer**
*Any specific actions or notes on review?*

# Testing
*Any relevant testing information and pipeline runs.*

# Checklist (please delete before completing or setting auto-complete)
- [x] Story Work items associated (not Tasks)
- [ ] Successful testing run(s) link provided
- [x] Title pattern should be `{work item number}: {title}`
- [x] Description covers all the changes in the PR
- [ ] This PR contains documentation
- [ ] This PR contains tests

# **How does this PR make you feel**:
BabyStepsSmallStepsGIF

### CODE REVIEW

File: common/scripts/access-control/flexible-server/Grant-FlexibleServerDBAccess.ps1 [hunk 1]
```diff
@@ -0,0 +1,112 @@
...
+    Set-Variable -Name InformationPreference -Value Continue -Scope global
+}
+
+Write-Host "${functionName} started at $($startTime.ToString('u'))"
+Write-Debug "${functionName}:PostgresHost:$PostgresHost"
+Write-Debug "${functionName}:PostgresDatabase:$PostgresDatabase"
+Write-Debug "${functionName}:ServiceMIName:$ServiceMIName"
+Write-Debug "${functionName}:TeamMIName:$TeamMIName"
+Write-Debug "${functionName}:TeamMIClientId=$TeamMIClientId"
+Write-Debug "${functionName}:TeamMIFederatedTokenFile=$TeamMIFederatedTokenFile"
```
Comment (ppotturi) [inline common/scripts/access-control/flexible-server/Grant-FlexibleServerDBAccess.ps1:45]: Is it going to write any sensitive information?

File: common/scripts/access-control/flexible-server/Grant-FlexibleServerAccess.ps1 [hunk 2]
```diff
@@ -0,0 +1,153 @@
...
+
+    Write-Host "Acquiring Access Token..."
+    $accessToken = Get-AzAccessToken -ResourceUrl "https://ossrdbms-aad.database.windows.net"
+    $ENV:PGPASSWORD = $accessToken.Token
+    Write-Host "Access Token Acquired"
+
+    [string]$command = Get-SQLScriptToCreatePrincipal
+    Write-Debug "${functionName}:command=$command"
+    
+    [System.IO.FileInfo]$tempFile = [System.IO.Path]::GetTempFileName()
```
Comment (ppotturi) [inline common/scripts/access-control/flexible-server/Grant-FlexibleServerAccess.ps1:112]: Define meaningful names for tempFile and tempFile1 ?

File: common/scripts/access-control/flexible-server/Grant-FlexibleServerDBAccess.ps1 (hunk 1 above)
Comment (jeevankuduvaravindran) [reply common/scripts/access-control/flexible-server/Grant-FlexibleServerDBAccess.ps1]: @ppotturi done

File: common/scripts/access-control/flexible-server/Grant-FlexibleServerAccess.ps1 (hunk 2 above)
Comment (jeevankuduvaravindran) [reply common/scripts/access-control/flexible-server/Grant-FlexibleServerAccess.ps1]: @ppotturi done

**Review by ppotturi — APPROVED** [review]

**Review by kbabigumira — APPROVED** [review]

**Pull request merged**